send/VsupIfis.pm

```diff
@@ -14,7 +14,7 @@ use DBI;
 sub init_config($) {
 
 	my $filename = shift;
-	my $configPath = "/home/perun/perun-sync/$filename";
+	my $configPath = "/etc/perun/ifis/$filename";
```

Comment:
Is this config used by more services/scrips? If so, then its ok. Otherwise I think it should be in /etc/perun/services/vsup_ifis/$filename as config related to the service.

Reply:
this config is also used by cron jobs so I guess it is ok this way